Resources/ConfigPresets/WizardsDen/wizardsDen.toml

```diff
@@ -11,6 +11,7 @@ panic_bunker.enable_without_admins = true
 panic_bunker.show_reason = true
 panic_bunker.custom_reason = "You have not played on a Wizard's Den server long enough to connect to this server. Please play on Wizard's Den Lizard, Leviathan, or Farm Grass Hopper until you have more playtime."
 
+
 [infolinks]
```

Comment:
There shouldn't be another blank line here? The other sections just have one blank line between them.